public/app/features/datasources/components/EditDataSource.tsx

```diff
@@ -219,6 +219,11 @@ export function EditDataSourceView({
                     ...dataSource,
                     jsonData: { ...dataSource.jsonData, ...jsonData },
                   }),
+                setSecureJsonData: (secureJsonData) =>
```

Comment:
nit: this allows setting the secure json data but from the slack conversation it seems plugin devs might also need to know which properties on secureJsonData are already set. [L214](https://github.com/grafana/grafana/blob/a4a35f2d0d270afcbdc6c1b5178063eef9916201/public/app/features/datasources/components/EditDataSource.tsx#L214) suggests to me this isn't possible right now?

Reply:
Should be fixed in latest commit 👍🏻 Good catch